src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -12,11 +12,14 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "llvm/Support/Debug.h"
+
 #include "src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.hpp"
 #include "src/Dialect/Krnl/DialectBuilder.hpp"
 #include "src/Dialect/ONNX/ONNXOps/ShapeHelper.hpp"
 
 #define DEBUG 0 /* Log which functions are simdized. */
+#define DEBUG_TYPE "unaryfuse"
```

Comment:
Since this type is for the whole file, would be better to have a more general name, e.g. `element-wise-lowering`.

Reply:
agree